### PR TITLE
feature/plugin-registry

### LINKED ISSUE #60: Reader-specific documentation can be hard to discover
### Feature Description

The `bioio` repository should point to/link to/reference/copy documentation for the readers.

### Use Case

The `bioio-ome-zarr` reader has the following documentation in its README.md:
> If using an `s3://` path to access a public S3 bucket, the `BioImage` constructor must be given a dictionary with `anon: True` in the `fs_kwargs` argument.

This may be relevant to many readers, but consistency and coordination across all readers is challenging due to the many repositories.

### Solution

1. Option 1: `bioio`'s README could link to `bioio-ome-zarr`'s README
2. Option 2: the text from the `bioio-ome-zarr` example above could be copied into `bioio`'s README
3. Option 3: Centralize all documentation by updating all readers' READMEs to say "Documentation for this repository lives at https://github.com/bioio-devs/bioio/" and adding a section for each reader to the central `bioio` README

### PR DESCRIPTION
**Pull request by BrianWhitneyAI**
## Description

The purpose of this PR is to resolve #60 and #17 (partially), and add a registry of plug-ins to the readme. In creating this PR I noticed that the Documentation of some of the readers has expired, so before merge I need to get each of the readers docs up again.

### CODE REVIEW

**Comment by BrianWhitneyAI** [comment]
Update. We have decided not to support individual documentation for our core reader-plugins. any variance from bioio-base should be described in the reademe of the plugin repo. In the situation that a plugin has a high variance from the core it will need its own documentation page, however we do not have any of those so far.

**Comment by evamaxfield** [comment]
> Update. We have decided not to support individual documentation for our core reader-plugins. any variance from bioio-base should be described in the reademe of the plugin repo. In the situation that a plugin has a high variance from the core it will need its own documentation page, however we do not have any of those so far.

I would add basically this sentence to right above or below the table. Something along the lines:

> Each reader plugin should closely follow the specification laid out in `bioio-base`. As such, it is likely common that reader plugins won't distribution their own documentation and users should instead review [bioio_base.reader.Reader](https://bioio-devs.github.io/bioio-base/bioio_base.html#bioio_base.reader.Reader) for API documentation for the underlying Reader API. We encourage plugin authors to publish their own documentation if they change or include new features into their published image readers.

**Comment by BrianWhitneyAI** [comment]
> > Update. We have decided not to support individual documentation for our core reader-plugins. any variance from bioio-base should be described in the reademe of the plugin repo. In the situation that a plugin has a high variance from the core it will need its own documentation page, however we do not have any of those so far.
> 
> I would add basically this sentence to right above or below the table. Something along the lines:
> 
> > Each reader plugin should closely follow the specification laid out in `bioio-base`. As such, it is likely common that reader plugins won't distribution their own documentation and users should instead review [bioio_base.reader.Reader](https://bioio-devs.github.io/bioio-base/bioio_base.html#bioio_base.reader.Reader) for API documentation for the underlying Reader API. We encourage plugin authors to publish their own documentation if they change or include new features into their published image readers.

added in [7ca0b8e](https://github.com/bioio-devs/bioio/pull/62/commits/7ca0b8e31f76b6fc1a5c9c84d3ea519a1628da34)

**Review by evamaxfield — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -47,6 +47,33 @@ img.set_scene(1)
...
+| bioio-bioformats       | [Full List](https://github.com/bioio-devs/bioio-bioformats/blob/175399d10d64194adcc7a6048c7b7537591824de/bioio_bioformats/reader_metadata.py#L24) | [Repo](https://github.com/bioio-devs/bioio-bioformats)           |
+
+Each reader plugin should closely follow the specification laid out in `bioio-base`. As such, it is likely common that reader plugins won't distribution their own documentation and users should instead review  [`bioio_base.reader.Reader`](https://bioio-devs.github.io/bioio-base/bioio_base.html#bioio_base.reader.Reader) for API documentation for the underlying Reader API. We encourage plugin authors to publish their own documentation if they change or include new features into their published image readers.
 
 ## Issues
 [_Click here to view all open issues in bioio-devs organization at once_](https://github.com/search?q=user%3Abioio-devs+is%3Aissue+is%3Aopen&type=issues&ref=advsearch)
+
+
+
+
```
Comment (pgarrison) [inline README.md:79]: Extraneous whitespace?

File: README.md [hunk 2]
```diff
@@ -47,6 +47,33 @@ img.set_scene(1)
...
+| bioio-ome-zarr         | .zarr       | [Repo](https://github.com/bioio-devs/bioio-ome-zarr)           |
+| bioio-sldy             | .sldy, .dir | [Repo](https://github.com/bioio-devs/bioio-sldy)           |
+| bioio-tifffile         | .tif , .tiff| [Repo](https://github.com/bioio-devs/bioio-tifffile)           |
+| bioio-tiff-glob        | .tiff (glob)| [Repo](https://github.com/bioio-devs/bioio-tiff-glob)           |
+| bioio-bioformats       | [Full List](https://github.com/bioio-devs/bioio-bioformats/blob/175399d10d64194adcc7a6048c7b7537591824de/bioio_bioformats/reader_metadata.py#L24) | [Repo](https://github.com/bioio-devs/bioio-bioformats)           |
+
+Each reader plugin should closely follow the specification laid out in `bioio-base`. As such, it is likely common that reader plugins won't distribution their own documentation and users should instead review  [`bioio_base.reader.Reader`](https://bioio-devs.github.io/bioio-base/bioio_base.html#bioio_base.reader.Reader) for API documentation for the underlying Reader API. We encourage plugin authors to publish their own documentation if they change or include new features into their published image readers.
```
Comment (pgarrison) [inline README.md:72]: Typo: distribute

**Review by pgarrison — APPROVED** [review]

File: README.md (hunk 2 above)
Comment (BrianWhitneyAI) [reply README.md]: resolved in [f10df4b](https://github.com/bioio-devs/bioio/pull/62/commits/f10df4bd958d97c5d195c9b432a8e2001a5a1b42)

File: README.md (hunk 1 above)
Comment (BrianWhitneyAI) [reply README.md]: resolved in [f10df4b](https://github.com/bioio-devs/bioio/pull/62/commits/f10df4bd958d97c5d195c9b432a8e2001a5a1b42)

**Pull request merged**